app/models/ems_refresh/save_inventory_middleware.rb

```diff
@@ -1,139 +0,0 @@
-module EmsRefresh::SaveInventoryMiddleware
```

Comment:
we usually leave the old refresh as backup, but e.g. @durandom decide to go all in with Ansible. Safer play is to keep the old one around for at least one release though and drive the old/new refresh with configuration